docs/course.md

```diff
@@ -52,10 +52,12 @@ This file specifies basic information about the course:
     "uuid": "cef0cbf3-6458-4f13-a418-ee4d7e7505dd",
     "name": "TAM 212",
     "title": "Introductory Dynamics",
+    "comment": "The assessment set order used here will be the one shown within PrairieLearn",
     "assessmentSets": [
         {"abbreviation": "HW", "name": "Homework", "heading": "Homeworks", "color": "green1"},
         {"abbreviation": "E", "name": "Exam", "heading": "Exams", "color": "red1"}
     ],
+    "comment": "Topics should be thought of as chapters or sections in a textbook",
```

Comment:
While I like these examples, I don't think we can support multiple comments within the same object.

Reply:
resolved in 502ad59f3